src/controllers/FacilityItemController.ts

```diff
@@ -68,9 +68,17 @@ export class FacilityItemController {
         );
       }
 
+      const metadata = {
```

Comment:
This create function seems to indicate that it can only create an `item` whereby `type` is only `ItemType.SPACE`. What about `ItemType.OTHER`?